Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/aws-amplify/aws-appsync-realtime-client-ios.git",
         "state": {
           "branch": null,
-          "revision": "28f61d7ca362699941cc8844d34d082f1eb49602",
-          "version": "1.10.0"
+          "revision": "36ed209d55aacc127356a387b51cef5f94ec9ac7",
+          "version": "2.0.0"
```

Comment:
hmm i don't see the corresponding Package.swift changes

Reply:
i think we need to add another commit to bump to 2.0.0 here https://github.com/aws-amplify/amplify-ios/blob/dev-preview/Package.swift#L11-L15

Reply:
Updated the PR. fixes ARTC to latest version and build failures